packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/post.test.js

```diff
@@ -0,0 +1,7 @@
+import test from 'ava';
+import { getIncarnation } from './tools/vat-status.js';
+
+test(`Verify Zoe vat incarnatin`, async t => {
```

Comment:
nit: lowercase for test names. also there's a typo.
```suggestion
test(`verify Zoe vat incarnation`, async t => {
```